### PR TITLE
Execute

### PR DESCRIPTION
**Pull request by Jani-siv**

### CODE REVIEW

File: arm/debugger/devices/cpu/assembler/Execute.cpp [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+Execute::ldr_t1_lit(uint16_t data, std::shared_ptr<registers::Registers> &registers, std::shared_ptr<memory::Mmu> &mmu)
+{
+    uint8_t imm8 = (data&0x00FF);
+    data >>= 8;
+    uint8_t destination = (data&0x7);
+    uint32_t pcVal = registers->readRegister(PC);
+    uint32_t dataFromMemory = mmu->ReadData32((pcVal - (imm8) + pcVal + 0x1));
+    registers->writeRegister(destination,dataFromMemory);
+    registers->writeRegister(PC,(registers->readRegister(PC)+align));
+
```
Comment (mezi-0) [inline arm/debugger/devices/cpu/assembler/Execute.cpp:159]: (159) Extra line

File: arm/debugger/devices/cpu/assembler/Execute.hpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+namespace assembler {
+
+class Execute {
+public:
+    Execute()=default;
+    virtual ~Execute()=default;
+    void executeCommand(std::string command, uint32_t data,
+                        std::shared_ptr<registers::Registers>& registers, std::shared_ptr<memory::Mmu>& mmu);
+private:
+
```
Comment (mezi-0) [inline arm/debugger/devices/cpu/assembler/Execute.hpp:22]: (22) Extra line

**Review by mezi-0 — CHANGES_REQUESTED** [review]
Few extra lines in:
"cpu/assembler/Execute.cpp"
"cpu/assembler/Execure.hpp"

**Comment by Jani-siv** [comment]
done

**Review by mezi-0 — APPROVED** [review]

**Pull request merged**